gcc/config/riscv/riscv-builtins.cc

```diff
@@ -18,6 +18,7 @@ You should have received a copy of the GNU General Public License
 along with GCC; see the file COPYING3.  If not see
 <http://www.gnu.org/licenses/>.  */
 
+#include "config/riscv/riscv-opts.h"
```

Comment:
This one doesn't need to

Reply:
yes, the clang-format add it automatically. I should delete it.